rapid_response_xblock/static/js/src/rapid.js

```diff
@@ -234,6 +234,13 @@
       );
     }
 
+    function hasExpression(text) {
+      return _.some([
+        _.includes(text, "\("),
+        _.includes(text, "\)"),
+        _.includes(text, "$")
+      ], Boolean);
```

Comment:
I believe [`_.some`](https://underscorejs.org/#some) returns a boolean by default. Is this `Boolean` argument needed?

Reply:
removed `Boolean`